cluster/addons/fluentd-elasticsearch/kibana-image/Makefile

```diff
@@ -1,6 +1,6 @@
 .PHONY:	build push
 
-TAG = 1.0
+TAG = 1.1
```

Comment:
Scary-- if someone with the keys to the kubernetes dockerhub user (e.g. me) builds this makefile, we'll blow away the official image.